velox/dwio/dwrf/test/E2EFilterTestBase.cpp

```diff
@@ -0,0 +1,411 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/dwrf/test/E2EFilterTestBase.h"
+
+DEFINE_int32(timing_repeats, 0, "Count of repeats for timing filter tests");
+
+namespace facebook::velox::dwio::dwrf {
```

Comment:
@oerling Orri, Thank you for making the refactoring PRs! I assume this E2EFilterTestBase will be used for multiple formats including Parquet since the data and functions in this class are all format agnostic. However it's still in namespace velox::dwio::dwrf. Shall we move it velox::dwio::common?